File: 1137-height-checker/height-checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> expect;
        int n = heights.size();
        for (int i = 0; i < n; i++)
        {
            expect.push_back(heights[i]);
        }
        ranges::sort(expect);
        int count = 0;
        for (int i = 0; i < n; i++)
        {
            if (expect[i] != heights[i]) count++;
        }
        return count;
    }
};

File: 1137-height-checker/height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int n = heights.length;
        int[] expected = Arrays.copyOf(heights, n);

        Arrays.sort(expected);

        int cnt = 0;
        for (int i = 0; i < n; i++) {
            if (expected[i] != heights[i]) cnt++;
        }

        return cnt;
    }
}